src/crimson/os/seastore/lba_manager/btree/btree_lba_manager.cc

```diff
@@ -890,15 +891,16 @@ BtreeLBAManager::_update_mapping(
 	    iter,
 	    ret
 	  ).si_then([c, ret, nextent](auto iter) {
-	    // child-ptr may already be correct,
-	    // see LBAManager::update_mappings()
-	    if (nextent && !nextent->has_parent_tracker()) {
+	    if (nextent) {
```

Comment:
It seems that this is controversial with the above `assert(nextent==nullptr)`.

Reply:
Hm, looking at `BtreeLBAManager::update_refcount()`, it seems guaranteed that nextent here is nullptr.

So we can just drop this logic, right?

Reply:
Yeah, I think so:-)

Reply:
@xxhdx1985126 I was wrong.

Looks the above `assert(nextent==nullptr)` is in `if (ret.refcount == 0)` but here is the `else` case, and possible to be called by `update_mapping()` with a valid extent.

Reply:
Ah, yes.